tests/5.1/metadirective/test_metadirective_nothing.c

```diff
@@ -0,0 +1,53 @@
+//===--------------------- test_metadirective_nothing.c ---------------------===//
+//
+// OpenMP API Version 5.1 Nov 2020
+// 
+// Test for nothing directive within metadirectives. Runs a variety of
+// metadirectives that check if the nothing directive is properly rendered.
+//
+////===---------------------------------------------------------------------===//
+
+#include <omp.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include "ompvv.h"
+
+#define N 1024
+
+int metadirective() {
+
+   int errors = 0;
+   int base_threads = 0;
+   int threads = 0;
+
+   #pragma omp target map(from:base_threads,threads)  
+   {
+	   	#pragma omp atomic write
+	 	base_threads = omp_get_num_threads();
+      	 	
+		#pragma omp metadirective \
```

Comment:
device={kind(nohost)}

Reply:
Implemented and used to leverage a sort of "compile time" test